app/view/sdl/ResetTimeoutPopUp.js

```diff
@@ -291,7 +291,7 @@ SDL.ResetTimeoutPopUp = Em.ContainerView.create({
             self = this;
             timeoutExpired = [];
             this.resetTimeoutRPCs.forEach(function(method) {
-                if(0 == self.timeoutSeconds[method]) {
+                if(1 == self.timeoutSeconds[method]) {
```

Comment:
@GetmanetsIrina If it's fixed the issue 901, better to move hardcoded number value to const for more understanding what is 1